pkg/update/update_cockroach_version.go

```diff
@@ -98,8 +99,10 @@ func UpdateClusterCockroachVersion(
 			return errors.Wrapf(err, "setting downgrade option for major roll forward failed")
 		}
 	}
-
-	wantImage := fmt.Sprintf("%s:%s", update.WantImageName, update.WantVersion.Original())
+	var wantImage string = update.WantImageName
```

Comment:
Please add comments on what we are doing here

Reply:
done